lms/djangoapps/course_home_api/outline/v1/views.py

```diff
@@ -246,23 +249,39 @@ def get(self, request, *args, **kwargs):
             access_expiration = get_access_expiration_data(request.user, course_overview)
             cert_data = get_cert_data(request.user, course, enrollment.mode) if is_enrolled else None
 
-            # Only show the set course goal message for enrolled, unverified
-            # users in a course that allows for verified statuses.
-            is_already_verified = CourseEnrollment.is_enrolled_as_verified(request.user, course_key)
-            if not is_already_verified and has_course_goal_permission(request, course_key_string,
-                                                                      {'is_enrolled': is_enrolled}):
-                course_goals = {
-                    'goal_options': valid_course_goals_ordered(include_unsure=True),
-                    'selected_goal': None
-                }
-
-                selected_goal = get_course_goal(request.user, course_key)
-                if selected_goal:
-                    course_goals['selected_goal'] = {
-                        'key': selected_goal.goal_key,
-                        'text': get_course_goal_text(selected_goal.goal_key),
+            if COURSE_GOALS_NUMBER_OF_DAYS_GOALS.is_enabled():
+                if (is_enrolled and ENABLE_COURSE_GOALS.is_enabled(course_key)
+                        and settings.FEATURES.get('ENABLE_COURSE_GOALS')):
+
+                    course_goals = {
+                        'goal_options': NUMBER_OF_DAYS_OPTIONS,
```

Comment:
the previous version of goals passed the goal options to the front-end. 
it seems like it would be nice to also pass the numeric options to the front-end, although it wouldn't actually be fully dynamic on the front-end, since it has words/icons on the buttons in addition to the number

thoughts on whether I should still include the goal_options?

Reply:
I don't think you should include it. We don't need to / want to make the frontend dynamic like that. We'll just hardcode it. And it feels like it's not the backend's place to give presentation suggestions ot the frontend like that. You could imagine a different frontend with different ideas about what options to present. What does the backend care?